tests/50federation/30room-join.pl

```diff
@@ -116,11 +116,11 @@
 test "Inbound federation can receive room-join requests",
    requires => [ $main::OUTBOUND_CLIENT, $main::INBOUND_SERVER,
                  $main::HOMESERVER_INFO[0],
-                 room_fixture( requires_users => [ local_user_fixture() ] ),
+                 local_user_and_room_fixtures(),
                  federation_user_id_fixture() ],
 
    do => sub {
-      my ( $outbound_client, $inbound_server, $info, $room_id, $user_id ) = @_;
+      my ( $outbound_client, $inbound_server, $info, $creator, $room_id, $user_id ) = @_;
```

Comment:
Since `$creator` isn't actually used by the test, you can just assign it into `undef` instead to leave a hole